app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -5,6 +5,7 @@ module TwoFactorAuthenticatable
   NEED_AUTHENTICATION = 'need_two_factor_authentication'.freeze
   OTP_LENGTH = 6
   DIRECT_OTP_LENGTH = 6
+  PROOFING_DIRECT_OTP_LENGTH = 6
```

Comment:
Maybe not a discussion for here... but I thought we landed back on the proofing code *definitely* having to have higher entropy and the 2FA one able to be lower?